packages/inference/src/config.ts

```diff
@@ -0,0 +1,2 @@
+export const HF_HUB_URL = "https://huggingface.co";
```

Comment:
(nit) this constant is usually called `HF_ENDPOINT` in the Python ecosystem

Reply:
For consistency with the other JS packages we'll keep the current name